src/fu-daemon.c

```diff
@@ -1994,6 +1994,9 @@ fu_daemon_daemon_method_call(GDBusConnection *connection,
 
 		g_debug("Called %s", method_name);
 		if (!authenticate) {
+			/* if we cannot authenticate then, if the peer is not
+			 * inherently trusted, only return a non-sensitive
+			 * subset of the settings */
```

Comment:
maybe `if we cannot authenticate and the peer is not`

Reply:
yes, sounds good, I changed it accordingly.